### ISSUE TITLE
Wording tweak for copy users see when 'type' isn't in our database

### ISSUE THREAD

**Issue by lizheard1** [PlasticPatrol/PlasticPatrolWeb#130]
Users are still confused and don't realise that they can type in a type that is not in our list and still submit the photo.

The current wording is:
'Sorry we don't have this type in our database at the moment. Once you submit you're collection it will be manually approved'

Which I think is a bit wordy and confusing. They need to know that they can type in a new type and can still submit their collection.
Any ideas of what to change it to? Or if we need a message at all - ie they enter a new type, no tags appear, then they just enter the brand?

**Comment by munroTom** [comment]
I think getting rid of the text entirely is best, the end user won't care that it's not in our list and it messes with the page styling a little
